DiagonalTraverse.java

```diff
@@ -0,0 +1,43 @@
+//Time Complexity: O(n)
+//Space Complexity: O(n)
```

Comment:
check your space complexity. Output space does not count in complexity

Reply:
Updated, thank you